File: MediaData/src/main/java/com/xu/mediadata/entity/Artist.java
package com.xu.mediadatatest.entity;

/**
 * desc:
 * *
 * user: xujj
 * time: 2022/8/26 14:35
 **/
public class Artist {

    private long id;
    private String name;
    private int musicCount;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getMusicCount() {
        return musicCount;
    }

    public void setMusicCount(int musicCount) {
        this.musicCount = musicCount;
    }

    @Override
    public String toString() {
        return "Artist{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", musicCount=" + musicCount +
                '}';
    }
}

File: MediaData/src/main/java/com/tool/mediadata/entity/Artist.kt
package com.tool.mediadata.entity

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.util.UUID

/**
 * desc:
 **
 * user: xujj
 * time: 2023/4/14 17:14
 **/
@Parcelize
data class Artist(
    var id: Long,
    var name: String,
    var musicCount: Int
) : Parcelable {

    companion object {
        fun example(name: String = "xxx"): Artist {
            return Artist(-1L, name, 0)
        }
    }
}